ui/src/app/components/help-sidebar-icons.tsx

```diff
@@ -505,12 +528,19 @@ export const HelpSidebarIcons = (props: HelpSidebarIconsProps) => {
     iconConfig({ iconId, pageKey, criteria, loadingError }).showIcon()
   );
 
+  if (
+    enableGkeApp &&
+    WorkspacePermissionsUtil.canWrite(workspace.accessLevel)
+  ) {
+    keys.push('apps');
+  }
+
   if (WorkspacePermissionsUtil.canWrite(workspace.accessLevel)) {
-    keys.push('runtime', 'terminal');
+    keys.push('runtimeConfig', 'terminal');
```

Comment:
just suggestions: 
if (WorkspacePermissionsUtil.canWrite(workspace.accessLevel)) {
  if (enableGkeApp) {
keys.push('apps');
} else {
keys.push('runtimeConfig', 'terminal');
}
}

Reply:
we want to keep 'runtimeConfig' and 'terminal' in all cases

Reply:
Ahhhh ok